DICOM/src/Microsoft.Health.DeID.SharedLib/Settings/EncryptSetting.cs

```diff
@@ -0,0 +1,47 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.Health.Dicom.DeID.SharedLib.Exceptions;
+
+namespace Microsoft.Health.DeID.SharedLib.Settings
+{
+    public class EncryptSetting
+    {
+        public string EncryptKey { get; set; } = Guid.NewGuid().ToString("N");
```

Comment:
🔴 **INSIST**

Is it the right time to use a better representation for encrypt/hash key?
Essentially a key is binary data which can't be directly save/load as a string without transformation.
I personally think this should be fixed unless you have convincing business justifications.
Same thing applies to the key of cryptofunction.
Other reviewers can comment on this.

Reply:
Checked the docs and found that [AesCng](https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.aescng?view=net-5.0) has functions to generate keys

Reply:
[RNGCryptoServiceProvider Class](https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rngcryptoserviceprovider?redirectedfrom=MSDN&view=net-5.0) can also help.
Its `GetNonZeroBytes` method fills an array of bytes with a cryptographically strong sequence of random nonzero values.